style.css

```diff
@@ -326,7 +349,7 @@ button {
   text-align: center;
   font-size: 1rem;
   color: #ebebff;
-  width: 30vw;
+  width: 35vw;
```

Comment:
- [ ] `optional`
- [ ] Great job you are doing 👏 
- [ ] please consider adjusting the size of you contact to for in reference to the figma template.
- [ ] also adjust the `submit button`
- [ ] please screenshots below

- [ ] `figma template`
![image](https://user-images.githubusercontent.com/66230306/192993345-c680a0ba-6fb1-4fb9-926e-52da019ab5b5.png)

`your template`
![width](https://user-images.githubusercontent.com/66230306/192994521-e552282b-f71a-4e07-ab64-b226ca60cecc.png)